docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -94,6 +94,7 @@ API Changes
   The `weather` argument of the above methods is now the first, required
   positional argument and the `times` argument is kept as the second keyword
   argument for capability during the deprecation period.
+* Parameter `pvsystem.DC_MODEL_PARAMS` is renamed to `pvsystem._DC_MODEL_PARAMS`
```

Comment:
We do need to keep this one because it's changing from a public to a private attribute. Let's add another sentence like "Users should not rely on this dictionary's existence or structure."